service_manifest.yml

```diff
@@ -108,6 +108,12 @@ heuristics:
     filetype: '*'
     description: Embedded code was discovered in a file posing as a jQuery library
 
+  - heur_id: 5
+    name: Microsoft Support Diagnostic Tool found in location redirection
+    score: 500
```

Comment:
set the score 1 until it runs through prod and we are sure that there are no FPs